components/Chat/components/ChatBox/ChatBox.tsx

```diff
@@ -33,15 +33,15 @@ const ChatBox = () => {
               <MyMessage
                 message={JSON.parse(message.data)}
                 key={message}
-                connectionId={message.connectionId}
+                // connectionId={message.connectionId}
               />
             )
           }
           return (
             <OtherMessage
               message={JSON.parse(message.data)}
               key={message}
-              connectionId={message.connectionId}
+              // connectionId={message.connectionId}
```

Comment:
```suggestion
```

Reply:
please remove commented code

@crazzywizard do we need this?

Reply:
i pass that for identification only.